src/components/badges/badge.spec.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Badge, {COLOR, SIZE} from './Badge';
+import Text from '../text/Text';
+import {shallow, mount} from 'enzyme';
+
+test('render', () => {
+  const badge = shallow(
+    <Badge>Test</Badge>
+  );
+
+  expect(badge.hasClass('sg-badge')).toEqual(true);
+});
+
+test('size', () => {
+  const badge = shallow(
+    <Badge size={SIZE.SMALL}>Test</Badge>
+  );
+
+  expect(badge.hasClass('sg-badge--small')).toEqual(true);
+});
+
+test('default font size', () => {
+  const badge = mount(
```

Comment:
You can test if using Text component in render method. And then check if you use correct props, not directly checking text css class (it's part of Text.spec)